atef/widgets/config/run_active.py

```diff
@@ -155,7 +155,11 @@ def __init__(self, *args, data: PreparedValueToSignal, **kwargs):
         insert_widget(self.status_label, self.status_label_placeholder)
         self.name_label.setText(data.name)
         self.target_label.setText(data.signal.name)
-        self.value_label.setText(str(data.value))
+        enum_strs = getattr(data.signal, 'enum_strs')
+        if enum_strs:
+            self.value_label.setText(str(enum_strs[int(data.value)]))
```

Comment:
Though unlikely, it'd be good to guard against enum string list length mistmatches

That is to say:

```
try:
    self.value_label.setText(str(enum_strs[int(data.value)]))
except IndexError:
    self.value_label.setText(str(data.value))
```